File: src/main/java/gamesoldstoreprojkt/Model/CardDTO.java
package gamesoldstoreprojkt.Model;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class CardDTO {
    private String cardNumber;
    private String cardOwnerName;
    private String cardPassword;
    private Date expiryDate;
    private String cardType;

}

File: frontend/gamesoldfrontprojkt/src/app/components/model/CardDTO.ts
export class CardDTO {
    cardNumber: String;
    cardOwnerName: String;
    cardPassword: String;
    expiryDate: Date;
    cardType: String;

    constructor(receivedCardNumber: String, receivedCardOwnerName: String, receivedCardPassword: String, receivedExpiryDate: Date, receivedCardType: String){
        this.cardNumber = receivedCardNumber;
        this.cardOwnerName = receivedCardOwnerName;
        this.cardPassword = receivedCardPassword;
        this.expiryDate = receivedExpiryDate;
        this.cardType = receivedCardType;
    }

}